example/src/App.tsx

```diff
@@ -1184,7 +1193,16 @@ class App extends React.Component<{}, State> {
         </div> */}
         <div>
           Pagination:
-          <Pagination {...PaginationDefaultProps} pageSizeList={[3, 6, 9]} showSizeChanger={true} onChange={(page: number) => this.setState({ paginationCurrent: page })} current={this.state.paginationCurrent} total={25} />
+          <Pagination
+            {...PaginationDefaultProps}
+            pageSizeList={[3, 6, 9]}
```

Comment:
@csumudu Is that page size list hard coded?

Reply:
Yes, it's a example page with dummy data just to test the widget